src/components/date-picker/pickers/Picker.ts

```diff
@@ -101,38 +87,8 @@ export abstract class Picker {
         });
     }
 
-    private handleCloseForFlatpickr(selectedDates) {
```

Comment:
"Dead code"

This piece of code are not working on master and I don't see us fix it in a near future.